ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -846,6 +846,18 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
+- as: aws-ipi-ovn-efs-hypershift-mgmt-longduration-f7-destructive
```

Comment:
Currently only storage team have several cases need the `mgmt` `longduration` `destructive`, so we could consider reduce the frequency. Using `f999` or `f28` should be ok, for `f999` means we manually trigger it when some rebase/ related codes changed.
cc to @duanwei33